Chapter 3 revision (#268)
worldbank/d4di: chapters/research-design.tex

%-----------------------------------------------------------------------------------------------

\begin{fullwidth}
Research design is the process of structuring field work
-- both experimental design and data collection --

> that will answer a specific research question.

Reviewer: ```suggestion
to answer a specific research question.
```

Author: 567034e81d8481b4eb444ad05cfafb9de7a596c7

You don't need to be an expert in this,
and there are lots of good resources out there
that focus on designing interventions and evaluations
as well as on econometric approaches.

> This section will present a brief overview

Reviewer: I think we need to make a stronger statement here about the value added of this chapter; we say just before that there are lots of good resources available, need to say more explicitly here how this is different.

Author: Tried again in 11b91f55827c40ba40a012f22d4478109808c867

of the most common methods that are used in development research.
Specifically, we will introduce you to several ``causal inference'' methods
that are frequently used to estimate the impact of real development programs.
The intent is for you to obtain an understanding of
the way in which each method constructs treatment and control groups,
the data structures needed to estimate the corresponding effects,
and available code tools that will assist you in this process (the list, of course, is not exhaustive).

> Thinking throug your design before starting data work is important for several reasons.

Reviewer: ```suggestion
Thinking through your design before starting data work is important for several reasons.
```

If you do not know how to calculate the correct estimator for your study,

> you will not be able to assess the power of your research design.

Reviewer: ```suggestion
you will not be able to assess the statistical power of your research design.
```

Author: b62766dd6792136b9572b0cf3285df8ef4d01dc0

You will also be unable to make decisions in the field

> when you inevitable have to allocate scarce resources

Reviewer: ```suggestion
when you inevitably have to allocate scarce resources
```

These are the changes in outcomes attributable to the treatment.\cite{abadie2018econometric}
  \index{treatment effect}
The primary goal of research design is to establish \textbf{causal identification} for an effect.
Causal identification means establishing that a change in an input directly altered an outcome.
  \index{identification}
When a study is well-identified, then we can say with confidence
that our estimate of the treatment effect would,
with an infinite amount of data,
give us a precise estimate of that treatment effect.
Under this condition, we can proceed to draw evidence from the limited samples we have access to,
using statistical techniques to express the uncertainty of not having infinite data.
Without identification, we cannot say that the estimate would be accurate,
even with unlimited data, and therefore cannot attribute it to the treatment
in the small samples that we typically have access to.
Conversely, more data is not a substitute for a well-identified experimental design.
Therefore it is important to understand how exactly your study
identifies its estimate of treatment effects,
so you can calculate and interpret those estimates appropriately.

> All the study designs we discuss here use the \textbf{potential outcomes} framework

Reviewer: add sidenote or index for potential outcomes framework

Author: Done as citation in 8e32d1dcfad3a6336f46eccfd992adc55515be31

\textbf{experimental} designs, in which the research team
is directly responsible for creating the variation in treatment,
and \textbf{quasi-experimental} designs, in which the team
identifies a ``natural'' source of variation and uses it for identification.
Neither approach is implicitly better or worse,
and both are capable of achieving effect identification under different contexts.

%-----------------------------------------------------------------------------------------------
\subsection{Estimating treatment effects using control groups}

The key assumption behind estimating treatment effects is that every
person, facility, or village (or whatever the unit of intervention is)
has two possible states: their outcomes if they do not receive some treatment
and their outcomes if they do receive that treatment.
Each unit's treatment effect is the individual difference between these two states,
and the \textbf{average treatment effect (ATE)} is the average of all
individual differences across the potentially treated population.
  \index{average treatment effect}
This is the parameter that most research designs attempt to estimate.

> Their goal is to establish a ``counterfactual scenario'' for the treatment group

Reviewer: why is counterfactual scenario marked this way instead of bold as the other terms? 

also, add a sidenote and/or index for counterfactual.

Author: 6876047e5d9ad88115446120d2f7ae3446e81459

Author: 56941c406c46660c75cfd8a4ca7d3bc3423dae52

with which outcomes can be directly compared.
There are several resources that provide more or less mathematically intensive

> approaches to understanding how various methods to his.

Reviewer: ```suggestion
approaches to understanding how various methods do this.
```

Author: c4feeaa7cedb25667c71d00e5fdf9e0883a3e3ef

\textit{Impact Evaluation in Practice} is a strong general guide to these methods.\sidenote{
  \url{https://www.worldbank.org/en/programs/sief-trust-fund/publication/impact-evaluation-in-practice}}
\textit{Causal Inference} and \textit{Causal Inference: The Mixtape}

> provides more detailed mathematical approaches fo the tools.\sidenote{

Author: 3fef5ea41bffafbec1e7c823026e7724a78be202

\textbf{Causal inference} methods are those in which we are able to estimate the
average treatment effect without observing individual-level effects,
but through some comparison of averages with a \textbf{control} group.
  \index{causal inference}\index{control group}
Every research design is based on a way of comparing another set of observations --
the ``control'' observations -- against the treatment group.
They all work to establish that the control observations would have been
identical \textit{on average} to the treated group in the absence of the treatment.
Then, the mathematical properties of averages imply that the calculated
difference in averages is equivalent to the average difference:
exactly the parameter we are seeking to estimate.
Therefore, almost all designs can be accurately described
as a series of between-group comparisons.\sidenote{
  \url{http://nickchk.com/econ305.html}}

Most of the methods that you will encounter rely on some variant of this strategy,
which is designed to maximize their ability to estimate the effect
of an average unit being offered the treatment you want to evaluate.
The focus on identification of the treatment effect, however,

> means there are several essential features to causal identification methods

Reviewer: ```suggestion
means there are several essential features of causal identification methods
```

that are not common in other types of statistical and data science work.
First, the econometric models and estimating equations used
do not attempt to create a predictive or comprehensive model
of how the outcome of interest is generated.
Typically, causal inference designs are not interested in predictive accuracy,
and the estimates and predictions that they produce
will not be as good at predicting outcomes or fitting the data as other models.
Additionally, when control variables or other variables are used in estimation,
there is no guarantee that the resulting parameters are marginal effects.
They can only be interpreted as correlative averages,
unless there are additional sources of identification.
The models you will construct and estimate are intended to do exactly one thing:
to express the intention of your project's research design,
and to accurately estimate the effect of the treatment it is evaluating.
In other words, these models tell the story of the research design
in a way that clarifies the exact comparison being made between control and treatment.

%-----------------------------------------------------------------------------------------------

> \subsection{Experimental and quasi-experimental research designs}

Author: I want to give a 30,000-foot view of causal inference methods to RAs. Usually they've done a lot of math but have no intuition for how these methods really work, or what that means in terms of data tracking or field tradeoffs. It's definitely less data oriented in a direct sense, but there is no point letting someone do data work who doesn't fully understand what is in this section. Maybe that needs to be explained better or...?

> \subsection{Experimental and quasi-experimental research designs}

Reviewer 2: moved to issue #333

Experimental research designs explicitly allow the research team
to change the condition of the populations being studied,\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Experimental_Methods}}

> often in the form of NGO programs, government regulations,

Reviewer: ```suggestion
often in the form of government programs, NGO projects, new regulations,
```

information campaigns, and many more types of interventions.\cite{banerjee2009experimental}

> The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{

Reviewer: ```suggestion
The classic experimental method is the \textbf{randomized control trial (RCT)}.\sidenote{
```

  \url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
  \index{randomized control trials}

> In randomized control trials, the control group is randomized --

Reviewer: This is perhaps just a matter of perspective, but i think this is more intuitive from the opposite angle, i.e. to say: the 'treatment' being studied is randomly allocated-- that is, from an eligible population, a random subset of units are given access to the treatment. The non-selected units serve as a counterfactual.

Author: I think you are right that it is more intuitive the other way, but I described it this way because I think it is important to take a moment to understand that this is _why_. I want people to stop and think while reading this section -- thinking of it as selecting the counterfactual at random from the eligible is a nice brain stretch IMO

that is, from an eligible population,
a random subset of units are not given access to the treatment,
so that they may serve as a counterfactual for those who are.
A randomized control group, intuitively, is meant to represent
how things would have turned out for the treated group
if they had not been treated, and it is particularly effective at doing so
as evidenced by its broad credibility in fields ranging from clinical medicine to development.
Therefore RCTs are very popular tools for determining the causal impact
of specific programs or policy interventions.

> However, there are many types of treatments that are impractical or unethical

Reviewer: ```suggestion
However, there are many other types of interventions that are impractical or unethical
```

to effectively approach using an experimental strategy,

> and therefore many limitations to accessing ``big questions''

Reviewer: ```suggestion
and therefore there are limitations to accessing ``big questions''
```

through RCT approaches.\sidenote{
  \url{https://www.nber.org/papers/w14690.pdf}}

Randomized designs all share several major statistical concerns.
The first is the fact that it is always possible to select a control group,
by chance, which is not in fact very similar to the treatment group.

> This feature is called randomization noise, and all RCTs share the need to assess

Author: I think it is? Added note in 8e20d2c95904914d245d7a459e9ea9286e110102

how randomization noise may impact the estimates that are obtained.
Second, takeup and implementation fidelity are extremely important,
since programs will by definition have no effect
if the population intended to be treated
does not accept or does not receive the treatment.

> Unfortunately, the loss of power happens with relatively small implementation gaps and is highly nonlinear:

Reviewer: ```suggestion
Unfortunately, the loss of statistical power that happens with relatively small implementation gaps is highly nonlinear:
```

Author: Slight loss of meaning as suggested. Revised for clarity: e704e2ae2784773c6a89b52a376664706dbe7950

that a researcher would attempt to create with an experiment.\cite{dinardo2016natural}
Unlike carefully planned experimental designs,
quasi-experimental designs typically require the extra luck
of having access to data collected at the right times and places
to exploit events that occurred in the past,
or having the ability to collect data in a time and place
where an event that produces causal identification occurred.
Therefore, these methods often use either secondary data,
or they use primary data in a cross-sectional retrospective method.

Quasi-experimental designs therefore can access a much broader range of questions,
and with much less effort in terms of executing an intervention.
However, they require in-depth understanding of the precise events
the researcher wishes to address in order to know what data to use
and how to model the underlying natural experiment.
Additionally, because the population exposed
to such events is limited by the scale of the event,
quasi-experimental designs are often power-constrained.
Since the research team cannot change the population of the study
or the treatment assignment, power is typically maximized by ensuring

> that sampling for data collection is carefully powered

Author: more power is obtained by becoming more powerful, obviously 4198e355e9afa0d38a4063775a359fb4f505aae2

and that attrition from the sampled groups is minimized.

%-----------------------------------------------------------------------------------------------
%-----------------------------------------------------------------------------------------------

> \section{Obtaining treatment effects from specific research designs}

Author: basically, "why some common designs give valid estimates". not sure how to reword at the moment...

%-----------------------------------------------------------------------------------------------
\subsection{Cross-sectional designs}

In an RCT, the control group is randomly constructed
from the population that is eligible to receive each treatment.
In an observational study, we present other evidence that a similar equivalence holds.
Therefore, by construction, each unit's receipt of the treatment
is unrelated to any of its other characteristics
and the ordinary least squares (OLS) regression
of outcome on treatment, without any control variables,

> is an unbiased estimate of the average treatment effect.

Author: I dodged "RCT" away from any particular section since all of the methods can be RCT ("experimental") or non-RCT ("quasi-experimental"). This is just to describe how a cross-sectional design works whether or not it is explicitly randomized. Re-arranged the paragraph to be much clearer about this. feadab6eb226d8ae425f3bcb41f5eef9ee1ee893

> A \textbf{cross-section} is the simplest data structure that can be used.

Author: I don't agree with that? You can make a case about a particular event or treatment being as good as random even if you have no pre-data. You'll often need some other type of data to support that but it doesn't have to be pre-intervention from your population. (Typically most applicable in the natural disaster literature where you can only do post-event surveys)

This type of data is easy to collect and handle because
you do not need track individual across time or across data sets.
A cross-section is simply a representative set of observations

> taken at a single point in time.

Author: This and more in feadab6eb226d8ae425f3bcb41f5eef9ee1ee893

If this point in time is after a treatment has been fully delivered,
then the outcome values at that point in time
already reflect the effect of the treatment.

For cross-sectional RCTs, what needs to be carefully maintained in data
is the treatment randomization process itself,
as well as detailed field data about differences
in data quality and loss to follow-up across groups.\cite{athey2017econometrics}
Only these details are needed to construct the appropriate estimator:
clustering of the estimate is required at the level
at which the treatment is assigned to observations,
and controls are required for variables which

> were used to stratify the treatment (in the form of strata fixed effects).\sidenote{

Author: fa3b30dc02dbadeb500a71614ccea451f5871d82

  \url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}}
\textbf{Randomization inference} can be used
to estimate the underlying variability in the randomization process
(more on this in the next chapter).
\textbf{Balance checks}\sidenote{

>   \textbf{Balance checks:} Statistical tests of the similarity of treatment and control groups.}

Author: This does not need to be the case. You can always check balance using other characteristics to make some assertion about the equivalence of groups even in a single period (https://www.sciencedirect.com/science/article/pii/S2214109X19300312)

>   \textbf{Balance checks:} Statistical tests of the similarity of treatment and control groups.}

Reviewer 2: moved to issue #337

are often reported as evidence of an effective randomization,
and are particularly important when the design is quasi-experimental
(since then the randomization process cannot be simulated explicitly).
However, controls for balance variables are usually unnecessary in RCTs,
because it is certain that the true data-generating process
has no correlation between the treatment and the balance factors.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments}}

Analysis is typically straightforward \textit{once you have a strong understanding of the randomization}.
A typical analysis will include a description of the sampling and randomization process,
summary statistics for the eligible population,
balance checks for randomization and sample selection,
a primary regression specification (with multiple hypotheses appropriately adjusted),
additional specifications with adjustments for non-response, balance, and other potential contamination,
and randomization-inference analysis or other placebo regression approaches.
There are a number of tools that are also available

> to help with the complete process of data collection,\sidenote{

Author: This refers to  the bcstats and hfc packages -- there is more but the intent is to have people use code tools during data collection

> to help with the complete process of data collection,\sidenote{

Author: 43b19d12da22e76d861f6eb3c004a4decf6f60d0

  \url{https://toolkit.povertyactionlab.org/resource/coding-resources-randomized-evaluations}}
to analyze balance,\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/iebaltab}}
and to visualize treatment effects.\sidenote{

>   \url{https://dimewiki.worldbank.org/wiki/iegraph}}

Author: I'd say in the wiki?

>   \url{https://dimewiki.worldbank.org/wiki/iegraph}}

Reviewer 3: This discussion in chapter 6

  \url{https://www.princeton.edu/~otorres/DID101.pdf}}
The estimated treatment effect is the excess growth
of units that receive the treatment, in the period they receive it:
calculating that value is equivalent to taking
the difference in means at endline and subtracting
the difference in means at baseline
(hence the singular ``difference-in-differences'').\cite{mckenzie2012beyond}
The regression model includes a control variable for treatment assignment,
and a control variable for time period,
but the treatment effect estimate corresponds to
an interaction variable for treatment and time:
it indicates the group of observations for which the treatment is active.
This model critically depends on the assumption that,
in the absense of the treatment,
the outcome of the two groups would have changed at the same rate over time,
typically referred to as the \textbf{parallel trends} assumption.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/often-unspoken-assumptions-behind-difference-difference-estimator-practice}}
Experimental approaches satisfy this requirement in expectation,
but a given randomization should still be checked for pre-trends

> as an extension of balance checking.

Author: ?

> as an extension of balance checking.

Author: 43b19d12da22e76d861f6eb3c004a4decf6f60d0

are critically important to maintain alongside the survey results.
In panel data structures, we attempt to observe the exact same units
in different points in time, so that we see the same individuals
both before and after they have received treatment (or not).\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/what-are-we-estimating-when-we-estimate-difference-differences}}
This allows each unit's baseline outcome (the outcome before the intervention) to be used
as an additional control for its endline outcome (the last outcome observation in the data),
a \textbf{fixed effects} design often referred to as an ANCOVA model,
which can provide large increases in power and robustness.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/another-reason-prefer-ancova-dealing-changes-measurement-between-baseline-and-follow}}
When tracking individuals over time for this purpose,
maintaining sampling and tracking records is especially important,
because attrition and loss to follow-up will remove that unit's information
from all points in time, not just the one they are unobserved in.
Panel-style experiments therefore require a lot more effort in field work

> for studies that use survey data.\sidenote{

Author: yes fcf5a1b1710ecbe4ac5413050ea95e03a6db41b7

  \url{https://www.princeton.edu/~otorres/Panel101.pdf}}
Since baseline and endline may be far apart in time,
it is important to create careful records during the first round
so that follow-ups can be conducted with the same subjects,
and attrition across rounds can be properly taken into account.\sidenote{
  \url{http://blogs.worldbank.org/impactevaluations/dealing-attrition-field-experiments}}

> As with cross-sectional designs, this set of study designs is widespread.

Author: 9fcbb61e28a08c969a6484d75c078fa947a163bb

as well as randomization, selection, and attrition analysis.
In trials of this type, reporting experimental design and execution
using the CONSORT style is common in many disciplines
and will help you to track your data over time.\cite{schulz2010consort}

%-----------------------------------------------------------------------------------------------
\subsection{Regression discontinuity}

\textbf{Regression discontinuity (RD)} designs exploit sharp breaks or limits
in policy designs to separate a group of potentially eligible recipients
into comparable gorups of individuals who do and do not receive a treatment.\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Regression_Discontinuity}}
These designs differ from cross-sectional and diff-in-diff designs
in that the group eligible to receive treatment is not defined directly,
but instead created during the treatment implementation.
  \index{regression discontinuity}
In an RD design, there is typically some program or event
that has limited availability due to practical considerations or policy choices
and is therefore made available only to individuals who meet a certain threshold requirement.

> The intuition of this design is that there is an underlying \textbf{running variable}

Author: index 4250d95e57df49218b937c28809ff23a806be840

by the potential recipients.
If the running variable is time there are special considerations.\cite{hausman2018regression}

Regression discontinuity designs are, once implemented,
very similar in analysis to cross-sectional or difference-in-differences designs.
Depending on the data that is available,
the analytical approach will center on the comparison of individuals
who are narrowly on the inclusion side of the discontinuity,
compared against those who are narrowly on the exclusion side.\sidenote{
  \url{https://cattaneo.princeton.edu/books/Cattaneo-Idrobo-Titiunik_2019\_CUP-Vol1.pdf}}
The regression model will be identical to the matching research designs,
i.e., contingent on whether data has one or more time periods
and whether the same units are known to be observed repeatedly.
The treatment effect will be identified, however, by the addition of a control
for the running variable -- meaning that the treatment effect variable
will only be applicable for observations in a small window around the cutoff.
(Spatial discontinuity designs are handled a bit differently due to their multidimensionality.\sidenote{

>   \url{https://blogs.worldbank.org/impactevaluations/spatial-jumps}})

Author: 1a0adb619df6f3c13fc9a8c4e3854556ffd97e87

> In the RD model, the functional form of that control and the size of that window,

Author: fe3f4f2fdc9a42cc751777d80d9713f7e7e82ddc, 1af32e3de28d7398bd8899b9acbf6851b5b38a91, a04af4c4237cf3811eddc139b04d1741668ce053

often referred to as the choice of \textbf{bandwidth} for the design,
are the critical parameters for the result.\cite{calonico2019regression}
Therefore, RD analysis often includes extensive robustness checking
using a variety of both functional forms and bandwidths,

> as well as placebo testing for non-realized locations of the cutoff.

Author: 3bfbe88787430768198ebd409234ce704026d9c6

to have very high variances relative other methods,
particularly when the relationship between the instrument and the treatment is small.\cite{young2017consistency}
IV designs furthermore rely on strong but untestable assumptions
about the relationship between the instrument and the outcome.\cite{bound1995problems}
Therefore IV designs face intense scrutiny on the strength and exogeneity of the instrument,
and tests for sensitivity to alternative specifications and samples
are usually required with an instrumental variables analysis.
However, the method has special experimental cases that are significantly easier to assess:
for example, a randomized treatment \textit{assignment} can be used as an instrument
for the eventual uptake of the treatment itself,
especially in cases where uptake is expected to be low,
or in circumstances where the treatment is available
to those who are not specifically assigned to it (``encouragement designs'').

In practice, there are a variety of packages that can be used
to analyse data and report results from instrumental variables designs.
While the built-in Stata command \texttt{ivregress} will often be used

> to create the final results, these are not sufficient on their own.

Author: c391d74b640883d270b2769a0cdcc020aeeb4377

The \textbf{first stage} of the design should be extensively tested,
to demonstrate the strength of the relationship between
the instrument and the treatment variable being instrumented.\cite{stock2005weak}
This can be done using the \texttt{weakiv} and \texttt{weakivtest} commands.\sidenote{
  \url{https://www.carolinpflueger.com/WangPfluegerWeakivtest_20141202.pdf}}
Additionally, tests should be run that identify and exclude individual
observations or clusters that have extreme effects on the estimator,

> using customized bootstrap or leave-one-out approaches.

Author: It's the Young reference again, I didn't want to repeat it. But I will, since it's so important. ef3fc1d83a72c3ff57d37ede14d0f1de73496d5b

Finally, bounds can be constructed allowing for imperfections
in the exogeneity of the instrument using loosened assumptions,
particularly when the underlying instrument is not directly randomized.\sidenote{
  \url{http://www.damianclarke.net/research/papers/practicalIV-CM.pdf}}

%-----------------------------------------------------------------------------------------------
\subsection{Matching}

\textbf{Matching} methods use observable characteristics of individuals

> to directly construct treatment and control groups as similar as possible

Author: 07dd863f739c31f85262e54881e75aa8d5454f72

to each other, either before a randomization process
or after the collection of non-randomized data.\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Matching}}
  \index{matching}
Matching observations may be one-to-one or many-to-many;
in any case, the result of a matching process
is similar in concept to the use of randomization strata
in simple randomized control trials.
In this way, the method can be conceptualized
as averaging across the results of a large number of ``micro-experiments''
in which the randomized units are verifiably similar aside from the treatment.

When matching is performed before a randomization process,
it can be done on any observable characteristics,
including outcomes, if they are available.

> The randomization should then record an indicator for the matching group.

Author: 9c246ff0972bc5952d781771a1981c506d0862ce

This approach is stratification taken to its most extreme:
it reduces the number of potential randomizations dramatically
from the possible number that would be available
if the matching was not conducted,
and therefore reduces the variance caused by the study design.
When matching is done ex post in order to substitute for randomization,
it is based on the assertion that within the matched groups,
the assignment of treatment is as good as random.
However, since most matching models rely on a specific linear model,

> such as the typical \textbf{propensity score matching} estimator,

Author: 0e678e4d8bd80092f4d42a0d614e60c158df99ce

they are open to the criticism of ``specification searching'',
meaning that researchers can try different models of matching
until one, by chance, leads to the final result that was desired;
analytical approaches have shown that the better the fit of the matching model,
the more likely it is that it has arisen by chance and is therefore biased.\cite{king2019propensity}
Newer methods, such as \textbf{coarsened exact matching},\cite{iacus2012causal}
are designed to remove some of the dependence on linearity.
In all ex-post cases, pre-specification of the exact matching model
can prevent some of the potential criticisms on this front,

> but ex-post matching in general is not regarded as a strong approach.

Reviewer: ```suggestion
but ex-post matching in general is not regarded as a strong identification strategy.
```

or, in the case of propensity scoring and similar approaches,
weighting the data appropriately in order to balance the analytical samples on the selected variables.
The \texttt{teffects} suite in Stata provides a wide variety
of estimators and analytical tools for various designs.\sidenote{
  \url{https://ssc.wisc.edu/sscc/pubs/stata_psmatch.htm}}
The coarsened exact matching (\texttt{cem}) package applies the nonparametric approach.\sidenote{
  \url{https://gking.harvard.edu/files/gking/files/cem-stata.pdf}}
DIME's \texttt{iematch} command in the \texttt{ietoolkit} package produces matchings based on a single continuous matching variable.\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/iematch}}
In any of these cases, detailed reporting of the matching model is required,
including the resulting effective weights of observations,
since in some cases the lack of overlapping supports for treatment and control
mean that a large number of observations will be weighted near zero
and the estimated effect will be generated based on a subset of the data.

%-----------------------------------------------------------------------------------------------
\subsection{Synthetic controls}

> \textbf{Synthetic control} is a relatively newer method

Reviewer: ```suggestion
\textbf{Synthetic control} is a relatively new method
```

for the case when appropriate counterfactual individuals

> do not exist in reality and there are very few (often only one) treatment unit.\cite{abadie2015comparative}

Reviewer: ```suggestion
do not exist in reality and there are very few (often only one) treatment units.\cite{abadie2015comparative}
```

  \index{synthetic controls}

> For example, state- or national-level policy changes

Reviewer: i think of state/national level policy changes as classic examples of natural experiments!

Transport and other big infrastructure projects very commonly use synthetic controls, since few units and placement cannot be randomized

Author: Agreed? That's why they fall here. Maybe it should be clearer that it is for the _effect on the state as a single unit_ when the effects at lower levels can't be measured? But this is hard to express...

Author: 56941c406c46660c75cfd8a4ca7d3bc3423dae52

%-----------------------------------------------------------------------------------------------

\begin{fullwidth}
Research design is the process of structuring field work
-- both experimental design and data collection --

> that will answer a specific research question.

Author: Isn't this pretty close? Is there something specific missing?

> that will answer a specific research question.

Reviewer 3: Not missing, but there are a few extra things:
- "structuring _field work_"
- "_experimental_ design"

How about: 
"Research design is the process of defining the methods and data that be will be used to answer a specific research question"

> that will answer a specific research question.

Author: I like it. 567034e81d8481b4eb444ad05cfafb9de7a596c7

and there are lots of good resources out there
that focus on designing interventions and evaluations
as well as on econometric approaches.
This section will present a brief overview
of the most common methods that are used in development research.
Specifically, we will introduce you to several ``causal inference'' methods
that are frequently used to understand the impact of real development programs.
The intent is for you to obtain an understanding of
the way in which each method constructs treatment and control groups,
the data structures needed to estimate the corresponding effects,
and any available code tools that will assist you in this process.

This is important to understand before going into the field for several reasons.
If you do not understand how to calculate the correct estimator for your study,
you will not be able to assess the power of your research design.
You will also be unable to make tradeoffs in the field
when you inevitable have to allocate scarce resources
between tasks like maximizing sample size
and ensuring follow-up with specific individuals.
You will save a lot of time by understanding the way
your data needs to be organized and set up as it comes in

> before you will be able to calculate meaningful results.

Author: 81262b21221aa02d4ca1f0f101723b445cf3db0a

> Just as importantly, understanding each of these approaches

Author: 038ca8d555058ec57551c30edaf13e79874a6b12

will allow you to keep your eyes open for research opportunities:
many of the most interesting projects occur because people in the field
recognize the opportunity to implement one of these methods on the fly
in response to an unexpected event in the field.
While somewhat more conceptual than practical,
a basic understanding of your project's chosen approach will make you

> much more effective at the analytical part of your work.

Author: 4c06288ec5465798110b434b18b0f9c97a2e47f4

> much more effective at the analytical part of your work.

Reviewer 3: I think this is the wrong commit

\end{fullwidth}

%-----------------------------------------------------------------------------------------------
%-----------------------------------------------------------------------------------------------

\section{Causality, inference, and identification}

The primary goal of research design is to establish \textbf{identification}
for a parameter of interest -- that is, to demonstrate
a source of variation in a particular input that has no other possible channel
to alter a particular outcome, in order to assert that some change in that outcome
was caused by that change in the input.

>   \index{identification}

Author: A bit better here? I still wanted to use input to stay abstract but directly linked input to treatment in the next sentence 038ca8d555058ec57551c30edaf13e79874a6b12

>   \index{identification}

Reviewer 3: Yes, that's great.

When we are discussing the types of inputs commonly referred to as
``programs'' or ``interventions'', we are typically attempting to obtain estimates
of a program-specific \textbf{treatment effect}, or the change in outcomes
directly attributable to exposure to what we call the \textbf{treatment}.\cite{abadie2018econometric}

>   \index{treatment effect}

Author: 4c06288ec5465798110b434b18b0f9c97a2e47f4

Under this condition, we can proceed to draw evidence from the limited samples we have access to,
using statistical techniques to express the uncertainty of not having infinite data.
Without identification, we cannot say that the estimate would be accurate,
even with unlimited data, and therefore cannot associate it to the treatment
in the small samples that we typically have access to.
Conversely, more data is not a substitute for a well-identified experimental design.
Therefore it is important to understand how exactly your study
identifies its estimate of treatment effects,
so you can calculate and interpret those estimates appropriately.
All the study designs we discuss here use the \textbf{potential outcomes} framework
to compare a group that received some treatment to another, counterfactual group.
Each of these types of approaches can be used in two contexts:
\textbf{experimental} designs, in which the research team
is directly responsible for creating the variation in treatment,
and \textbf{quasi-experimental} designs, in which the team
identifies a ``natural'' source of variation and uses it for identification.
Neither type of approach is implicitly better or worse,

> and both are capable of achieving effect identification under different contexts.

Author: "experimental" and "quasi"

> and both are capable of achieving effect identification under different contexts.

Reviewer 3: ok, I think I was confused because in line 67 "approaches" refers to the different methods in the chapter, and in line 72, "type of approach" it refers to "contexts", except that "contexts" is used again in line 77 meaning something else.

How about
- "each of these designs can be used in two settings" in line 67
- "neither setting" in line 76

> and both are capable of achieving effect identification under different contexts.

Author: fa922de7df148f33f51d430a250f2f57c2fac5ec

Each unit's treatment effect is the individual difference between these two states,
and the \textbf{average treatment effect (ATE)} is the average of all of
these differences across the potentially treated population.
  \index{average treatment effect}
This is the most common parameter that research designs will want to estimate.
In most designs, the goal is to establish a ``counterfactual scenario'' for the treatment group
with which outcomes can be directly compared.
There are several resources that provide more or less mathematically intensive
approaches to understanding how various methods to his.
\textit{Impact Evaluation in Practice} is a strong general guide to these methods.\sidenote{
  \url{https://www.worldbank.org/en/programs/sief-trust-fund/publication/impact-evaluation-in-practice}}
\textit{Causal Inference} and \textit{Causal Inference: The Mixtape}
provides more detailed mathematical approaches fo the tools.\sidenote{
  \url{https://www.hsph.harvard.edu/miguel-hernan/causal-inference-book/}
  \\ \noindent \url{http://scunning.com/cunningham_mixtape.pdf}}
\textit{Mostly Harmless Econometrics} and \textit{Mastering Metrics}

> are canonical treatments of the statistical principles behind all econometric approaches.\sidenote{

Author: 36fbd1d1f67e8a72695dcadfacdb330a75c5dff6

\textbf{Causal inference} methods are those in which we are able to estimate the
average treatment effect without observing individual-level effects,
but can obtain it from some comparison of averages with a \textbf{control} group.
  \index{causal inference}\index{control group}
Every research design is based around a way of comparing another set of observations --
the ``control'' observations -- against the treatment group.
They all work to establish that the control observations would have been
identical \textit{on average} to the treated group in the absence of the treatment.
Then, the mathematical properties of averages implies that the calculated
difference in averages is equivalent to the average difference:
exactly the parameter we are seeking to estimate.
Therefore, almost all designs can be accurately described
as a series of between-group comparisons.\sidenote{
  \url{http://nickchk.com/econ305.html}}

Most of the methods that you will encounter rely on some variant of this strategy,
which is designed to maximize the ability to estimate the effect
of an average unit being offered the treatment being evaluated.
The focus on identification of the treatment effect, however,

> means there are several essential features to this approach

Author: a250a1cb110ace99d7e145c98d7042f1b48eda3f

that are not common in other types of statistical and data science work.
First, the econometric models and estimating equations used
do not attempt to create a predictive or comprehensive model
of how the outcome of interest is generated.
Typically, these designs are not interested in predictive accuracy,
and the estimates and predictions that these models produce
will not be as good at predicting outcomes or fitting the data as other models.
Additionally, when control variables or other variables are used in estimation,
there is no guarantee that those parameters are marginal effects.

> They can only be interpreted as correlative averages,

Author: Correlative is correct here I think

unless the experimenter has additional sources of identification for them.
The models you will construct and estimate are intended to do exactly one thing:
to express the intention of your project's research design,
and to accurately estimate the effect of the treatment it is evaluating.
In other words, these models tell the story of the research design
in a way that clarifies the exact comparison being made between control and treatment.

%-----------------------------------------------------------------------------------------------
\subsection{Experimental and quasi-experimental research designs}

Experimental research designs explicitly allow the research team
to change the condition of the populations being studied,\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Experimental_Methods}}
often in the form of NGO programs, government regulations,
information campaigns, and many more types of interventions.\cite{banerjee2009experimental}

> The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{

Author: Classic experimental causal inference method

> The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{

Reviewer 3: ```suggestion
The classic method for experimental causal inference is the \textbf{randomized control trial (RCT)}.\sidenote{
```

> The classic method is the \textbf{randomized control trial (RCT)}.\sidenote{

Author: fa922de7df148f33f51d430a250f2f57c2fac5ec

  \url{https://dimewiki.worldbank.org/wiki/Randomized_Control_Trials}}
  \index{randomized control trials}
In randomized control trials, the control group is randomized --
that is, from an eligible population,
a random subset of units are not given access to the treatment,
so that they may serve as a counterfactual for those who are.
A randomized control group, intuitively, is meant to represent
how things would have turned out for the treated group
if they had not been treated, and it is particularly effective at doing so

> as evidenced by its broad credibility in fields ranging from clinical medicine to development.

Author: really??

> as evidenced by its broad credibility in fields ranging from clinical medicine to development.

Reviewer: i don't think a citation is necessary here

> as evidenced by its broad credibility in fields ranging from clinical medicine to development.

Reviewer 3: I agree that RCT has broad credibility in many fields, but in a quick google search, I couldn't find a  source that shows that RCTs are more likely to be published in any field. I also found a bunch of articles and paper saying that RCTs are popular in many fields, but they all seem to rely on the assumption that this is an undisputed fact rather than showing any data to corroborate that. It may be asking too much of this book to do so when no one elses seems to do it, but I'd like to see some numbers.

> as evidenced by its broad credibility in fields ranging from clinical medicine to development.

Reviewer 2: moved to issue #335

Therefore RCTs are very popular tools for determining the causal impact

> of specific programs or policy interventions.

Author: don't make me do this https://www.nobelprize.org/prizes/economic-sciences/2019/ceremony-speech/

> of specific programs or policy interventions.

Reviewer: @luizaandrade let's discuss but i think this is a reasonably general point. though reference to the nobel prize could be fitting!

> of specific programs or policy interventions.

Author: 43b19d12da22e76d861f6eb3c004a4decf6f60d0

However, there are many types of treatments that are impractical or unethical
to effectively approach using an experimental strategy,
and therefore many limitations to accessing ``big questions''
through RCT approaches.\sidenote{
  \url{https://www.nber.org/papers/w14690.pdf}}

Randomized designs all share several major statistical concerns.
The first is the fact that it is always possible to select a control group,
by chance, which was not in fact going to be very similar to the treatment group.
This feature is called randomization noise, and all RCTs share the need to understand
how randomization noise may impact the estimates that are obtained.
Second, takeup and implementation fidelity are extremely important,
since programs will by definition have no effect
if they are not in fact accepted by or delivered to

> the people who are supposed to receive them.

Author: a520f5e8e436cf89b06f1a49269c783123406872

> Unfortunately, these effects kick in very quickly and are highly nonlinear:

Author: 747140b0b696728625e2942bf9d6aef3fd42f22e

Such effects are also very hard to correct ex post,
since they require strong assumptions about the randomness or non-randomness of takeup.
Therefore a large amount of field time and descriptive work
must be dedicated to understanding how these effects played out in a given study,
and often overshadow the effort put into the econometric design itself.

\textbf{Quasi-experimental} research designs,\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Quasi-Experimental_Methods}}
by contrast, are inference methods based on events not controlled by the research team.
Instead, they rely on ``experiments of nature'',
in which natural variation can be argued to approximate
the type of exogenous variation in treatment availability
that a researcher would attempt to create with an experiment.\cite{dinardo2016natural}
Unlike with carefully planned experimental designs,
quasi-experimental designs typically require the extra luck
of having access to data collected at the right times and places
to exploit events that occurred in the past,
or having the ability to collect data in a time and place

> dictated by the availability of identification.

Author: fa9da15997a74244ff6bac941a5e10bd4cdb140c

Therefore, these methods often use either secondary data,

> or use primary data in a cross-sectional retrospective method.

Author: fec36b11278beab3624d879e848c86d1df21fefd

Quasi-experimental designs therefore can access a much broader range of questions,
and with much less effort in terms of executing an intervention.
However, they require in-depth understanding of the precise events
the researcher wishes to address in order to know what data to collect
and how to model the underlying natural experiment.
Additionally, because the population who will have been exposed
to such events is limited by the scale of the event,
quasi-experimental designs are often power-constrained.
There is nothing the research team can do to increase power
by providing treatment to more people or expanding the control group:
instead, power is typically maximized by ensuring
that sampling is carried out effectively

> and that attrition from the sampled groups is dealt with effectively.

Author: 56c0aded46e95fdb3ef39dcb12b362248ab85c5d

> and that attrition from the sampled groups is dealt with effectively.

Reviewer 3: > _power_ is typically maximized by ensuring
> that sampling for data collection is carefully _powered_

I have zero experience with power calcs for quasi-experimental designs. How do you ensure that sampling is powered?

> and that attrition from the sampled groups is dealt with effectively.

Author: Made more clear in another response. Basically by not spreading your sampling too thin over multiple hypotheses since you have a fixed population to draw from

Sampling noise and survey non-response are therefore analogous
to the randomization noise and implementation failures

> that can be observed in RCT designs, and have similar implications for field work.

Author: True. removed. 682436bb0784e348fa6bade381afb90117632117

%-----------------------------------------------------------------------------------------------
%-----------------------------------------------------------------------------------------------
\section{Obtaining treatment effects from specific research designs}

%-----------------------------------------------------------------------------------------------
\subsection{Cross-sectional designs}

\textbf{Cross-sectional} surveys are the simplest possible study design:

> a program is implemented, surveys are conducted, and data is analyzed.

Author: k

for research teams do not need track anything over time.
A cross-section is simply a representative set of observations
taken at a single point in time.
If this point in time is after a treatment has been fully delivered,
then the outcome values at that point in time
already reflect the effect of the treatment.

What needs to be carefully maintained in data for cross-sectional RCTs
is the treatment randomization process itself,
as well as detailed field data about differences
in data quality and loss-to-follow-up across groups.\cite{athey2017econometrics}
Only these details are needed to construct the appropriate estimator:
clustering of the estimate is required at the level
at which the treatment is assigned to observations,
and controls are required for variables which
were used to stratify the treatment (in the form of strata fixed effects).\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}}
\textbf{Randomization inference} can be used
to esetimate the underlying variability in the randomization process
(more on this in the next chapter).

> \textbf{Balance checks} are often reported as evidence of an effective randomization,

Author: 997d68366d974667dc79da2256a24d61167bb256

> \textbf{Balance checks} are often reported as evidence of an effective randomization,

Author: any suggestion on citation?

> \textbf{Balance checks} are often reported as evidence of an effective randomization,

Reviewer 3: https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments
?

> \textbf{Balance checks} are often reported as evidence of an effective randomization,

Reviewer 3: I like this one a lot

> \textbf{Balance checks} are often reported as evidence of an effective randomization,

Reviewer 3: it's actually already there, so marking this as resolved

> and are particularly important when the design is quasi-experimental

Author: Should I remove that from this section? I really don't want to list out for each design how these things differ since that is a general idea addressed in the previous and not intended to be the point of this section

> and are particularly important when the design is quasi-experimental

Reviewer 2: moved to issue #337

(since then the randomization process cannot be simulated explicitly).
However, controls for balance variables are usually unnecessary in RCTs,
because it is certain that the true data-generating process
has no correlation between the treatment and the balance factors.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments}}

Analysis is typically straightforward with a strong understanding of the randomization.

> A typical analysis will include a decription of the sampling and randomization process,

Author: I think these are the basics, no? what else would you add?

> A typical analysis will include a decription of the sampling and randomization process,

Reviewer 3: I mean describing the sampling and randomization processes go in a section of the paper that does not yet include data analysis. This will only be present in the data analysis through descriptive tables (summary stats, balance tests, etc).

What is not clear to me is why "description of the sampling and randomization process" is considered data analysis.

calculating that value is equivalent to taking
the difference in means at endline and subtracting
the difference in means at baseline
(hence the singular ``difference-in-differences'').\cite{mckenzie2012beyond}
The regression model includes a control variable for treatment assignment,
and a control variable for the measurement round,
but the treatment effect estimate corresponds to
an interaction variable for treatment and round:
the group of observations for which the treatment is active.
This model critically depends on the assumption that,
in the absense of the treatment,
the two groups would have changed performance at the same rate over time,
typically referred to as the \textbf{parallel trends} assumption.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/often-unspoken-assumptions-behind-difference-difference-estimator-practice}}
Experimental approaches satisfy this requirement in expectation,
but a given randomization should still be checked for pre-trends
as an extension of balance checking.

There are two main types of data structures for differences-in-differences:
\textbf{repeated cross-sections} and \textbf{panel data}.

> In repeated cross-sections, each round contains a random sample

Author: trying plain language context in 5bce9f1656adb2b8505703c2c7529569ae6d67bf

of observations from the treated and untreated groups;
as in cross-sectional designs, both the randomization and sampling processes,

> as well as their execution in the field,

Author: removed in f4bca7eaae979522de660bfdce4a83fc12354c5e

> are critically important to maintain alongside the survey results.

Author: removed in f4bca7eaae979522de660bfdce4a83fc12354c5e

In panel data structures, we attempt to observe the exact same units
in the repeated rounds, so that we see the same individuals
both before and after they have received treatment (or not).\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/what-are-we-estimating-when-we-estimate-difference-differences}}
This allows each unit's baseline outcome to be used
as an additional control for its endline outcome,
a \textbf{fixed effects} design often referred to as an ANCOVA model,
which can provide large increases in power and robustness.\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/another-reason-prefer-ancova-dealing-changes-measurement-between-baseline-and-follow}}
When tracking individuals over rounds for this purpose,
maintaining sampling and tracking records is especially important,

> because attrition and loss-to-follow-up will remove that unit's information

Author: failure to re-survey. might not be the same as attrition (dropping out of study). also fixed hyphenation in 081760ed6e67700001e443cdaada562cc4afa076

> because attrition and loss-to-follow-up will remove that unit's information

Reviewer 3: cool! learned a fancy new expression :D

%-----------------------------------------------------------------------------------------------
\subsection{Regression discontinuity}

\textbf{Regression discontinuity (RD)} designs exploit sharp breaks or limits
in policy designs to separate a group of potentially eligible recipients
into comparable gorups of individuals who do and do not recieve a treatment.\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/Regression_Discontinuity}}
These types of designs differ from cross-sectional and diff-in-diff designs
in that the group eligible to receive treatment is not defined directly,
but instead created during the process of the treatment implementation.
  \index{regression discontinuity}
In an RD design, there is typically some program or event
which has limited availability due to practical considerations or policy choices
and is therefore made available only to individuals who meet a certain threshold requirement.
The intuition of this design is that there is an underlying \textbf{running variable}
which serves as the sole determinant of access to the program,
and a strict cutoff determines the value of this variable at which eligibility stops.\cite{imbens2008regression}

> Common examples are test score thresholds, income thresholds, and some types of lotteries.\sidenote{

Author: Pricing lotteries / score-cutoff vouchers / auction style stuff, when the running variable is willingness to pay or some other eligibility? It can be

  \url{http://blogs.worldbank.org/impactevaluations/regression-discontinuity-porn}}

> The intuition is that individuals who are just on the recieving side of the threshold

Author: 70093639f8ab479d73883ff2ca4edd7389b2ddf9

will be very nearly indistinguishable from those on the non-receiving side,
and their post-treatment outcomes are therefore directly comparable.\cite{lee2010regression}
The key assumption here is that the running variable cannot be directly manipulated
by the potential recipients; if the running variable is time there are special considerations.\cite{hausman2018regression}

Regression discontinuity designs are, once implemented,
very similar in analysis to cross-sectional or differences-in-differences designs.
Depending on the data that is available,
the analytical approach will center on the comparison of individuals
who are narrowly on the inclusion side of the discontinuity,
compared against those who are narrowly on the exclusion side.\sidenote{
  \url{https://cattaneo.princeton.edu/books/Cattaneo-Idrobo-Titiunik_2019\_CUP-Vol1.pdf}}
The regression model will be identical to the matching research designs
(ie, contingent whether data has one or more rounds
and whether the same units are known to be observed repeatedly).
The treatment effect will be identified, however, by the addition of a control
for the running variable -- meaning that the treatment effect variable

> will only be applicable for observations in a small window around the cutoff.

Author: no, because it is also for those just below

> will only be applicable for observations in a small window around the cutoff.

Reviewer 3: Again I don't understand my own comment (I am trying to get better at this). I think what is confusing in the sentence is that is says "the treatment effect _variable_ will only be _applicable_ for observations ...".

So I think something like the below would address Maria's point as well

>The treatment effect will be identified, however, by the addition of a control
for the running variable -- meaning that this method only estimates the impact on the group of observations inside a small window around the cutoff.

> will only be applicable for observations in a small window around the cutoff.

Author: 04b2122c31534843121d4ee7b62e6fb7bdb36da5

(Spatial discontinuity designs are handled a bit differently due to their multidimensionality.\sidenote{

>   \url{https://blogs.worldbank.org/impactevaluations/spatial-jumps}})

Author: not sure what you mean exactly but that was a lot of parenthetical for one paragraph 7531b12e971b7a60d4b7f27e6bd89e5716a3c2a1

>   \url{https://blogs.worldbank.org/impactevaluations/spatial-jumps}})

Reviewer 3: I just mean that the line on spatial discontinuity and this line on time discontinuities could come together -- something like "time and space can both be used as running variables, but there are special considerations to be made in both cases".

https://github.com/worldbank/d4di/blob/4198e355e9afa0d38a4063775a359fb4f505aae2/chapters/research-design.tex#L392

>   \url{https://blogs.worldbank.org/impactevaluations/spatial-jumps}})

Author: yep 5b30a1ec618a8c03b819100cb3bf97cbb5a47285

These packages support the testing and reporting
of robust plotting and estimation procedures,
tests for manipulation of the running variable,
and tests for power, sample size, and randomization inference approaches
that will complement the main regression approach used for point estimates.

%-----------------------------------------------------------------------------------------------
\subsection{Instrumental variables}

\textbf{Instrumental variables (IV)} designs, unlike the previous approaches,
begin by assuming that the treatment delivered in the study in question is
linked to outcomes such that the effect is not directly identifiable.
Instead, similar to regression discontinuity designs,
IV attempts to focus on a subset of the variation in treatment uptake
and assesses that limited window of variation that can be argued
to be unrelated to other factors.\cite{angrist2001instrumental}
To do so, the IV approach selects an \textbf{instrument}
for the treatment status -- an otherwise-unrelated predictor of exposure to treatment
that affects the uptake status of an individual.\sidenote{
  \url{https://dimewiki.worldbank.org/wiki/instrumental_variables}}

> Whereas regression discontinuity designs are ``sharp'' --

Author: fuzzy RD is just IV, right?

> Whereas regression discontinuity designs are ``sharp'' --

Reviewer 3: one more thing learned today

in which the randomized units are verifiably similar aside from the treatment.

When matching is performed before a randomization process,
it can be done on any observable characteristics,
including outcomes, if they are available.
The randomization should then record an indicator for the matching group.
This approach is stratification taken to its most extreme:
it reduces the number of potential randomizations dramatically
from the possible number that would be available
if the matching was not conducted,
and therefore reduces the variance caused by the study design.
When matching is done ex post in order to substitute for randomization,
it is based on the assertion that within the matched groups,
the assignment of treatment is as good as random.
However, since most matching models rely on a specific linear model,
such as the typical \textbf{propensity score matching} estimator,
they are open to the criticism of ``specification searching'',
meaning that researchers can try different models of matching
until one, by chance, leads to the final result that was desired;
analytical approaches have shown that the better the fit of the matching model,

> the more likely it is that it has arisen by chance and is therefore biased.

Author: 1b32e5a56fa16307e0c5536a4ef0c25bc3a2f6bc

Newer methods, such as \textbf{coarsened exact matching},\cite{iacus2012causal}
are designed to remove some of the dependence on linearity.
In all ex-post cases, pre-specification of the exact matching model
can prevent some of the potential criticisms on this front,
but ex-post matching in general is not regarded as a strong approach.

Analysis of data from matching designs is relatively straightforward;
the simplest design only requires controls (indicator variables) for each group
or, in the case of propensity scoring and similar approaches,
weighting the data appropriately in order to balance the analytical samples on the selected variables.
The \texttt{teffects} suite in Stata provides a wide variety
of estimators and analytical tools for various designs.\sidenote{
  \url{https://ssc.wisc.edu/sscc/pubs/stata_psmatch.htm}}
The coarsened exact matching (\texttt{cem}) package applies the nonparametric approach.\sidenote{
  \url{https://gking.harvard.edu/files/gking/files/cem-stata.pdf}}
DIME's \texttt{iematch} command in the \texttt{ietoolkit} package produces matchings based on a single continuous matching variable.\sidenote{

>   \url{https://dimewiki.worldbank.org/wiki/iematch}}

Reviewer 2: It can, but it cannot create the p-score. What aspect of this did you not think it could be used for?

>   \url{https://dimewiki.worldbank.org/wiki/iematch}}

Reviewer: what does iematch do that the other commands do not? make a 1 line case for it here

>   \url{https://dimewiki.worldbank.org/wiki/iematch}}

Reviewer 2: nothing. 😄 That't true but that was also the purpose. When I wrote this command we needed to do a matching on a continuous variable. We had the variable already and we did not want to do do any estimations using the result of the matching. All the commands I found only did matching as a part of creating a p-score or as a part of an estimation. So the whole point is that is doing nothing but matching.

>   \url{https://dimewiki.worldbank.org/wiki/iematch}}

Reviewer 2: keep as is

\textbf{Synthetic control} is a relatively newer method
for the case when appropriate counterfactual individuals
do not exist in reality and there are very few (often only one) treatment unit.\cite{abadie2015comparative}
  \index{synthetic controls}
For example, state- or national-level policy changes
are typically very difficult to find valid comparators for,
since the set of potential comparators is usually small and diverse
and therefore there are no close matches to the treated unit.
Intuitively, the synthetic control method works
by constructing a counterfactual version of the treated unit
using an average of the other units available.\cite{abadie2010synthetic}
This is a particularly effective approach
when the lower-level components of the units would be directly comparable:
people, households, business, and so on in the case of states and countries;
or passengers or cargo shipments in the case of transport corridors, for example.\cite{gobillon2016regional}
This is because in those situations the average of the untreated units
can be thought of as balancing by matching the composition of the treated unit.

To construct this estimator, the synthetic controls method requires

> a significant amount of retrospective data on the treatment unit and possible comparators,

Author: ok bfdd5d0bde4304578c981537362c559e6fc68ad9

> a significant amount of retrospective data on the treatment unit and possible comparators,

Reviewer 3: I think we can leave it as "requires retrospective data" if that seems ok, and not get into the weeds of how many periods and how that affects the quality

> a significant amount of retrospective data on the treatment unit and possible comparators,

Author: d18cd39d9010ac6127197f7c1d3b80d0bb5daf9a

including historical data on the outcome of interest for all units.
The counterfactual blend is chosen by optimizing the prediction of past outcomes
based on the potential input characteristics,
and typically selects a small set of comparators to weight into the final analysis.
These datasets therefore may not have a large number of variables or observations,
but the extent of the time series both before and after the implementation

> of the treatment are the key sources of power for the estimate.

Author: 4721f3e4ab2ed2f98fb25b21291014b6a591221c

Visualizations are often excellent demonstrations of these results.

> The \texttt{synth} package provides functionality for use in Stata,

Author: This is what you meant right 4c06288ec5465798110b434b18b0f9c97a2e47f4
